drivers/volume/portworx/portworx.go

```diff
@@ -3168,6 +3168,49 @@ func (p *portworx) DeactivateClusterDomain(cdu *storkapi.ClusterDomainUpdate) er
 	return err
 }
 
+func (p *portworx) GetClusterDomainNodes() (map[string][]*api.Node, error) {
```

Comment:
This function needs UT. We can mock the clusterManager interface.

Reply:
Though clusterManager interface is mockable but to get it there should be a mockable `getClusterManagerClient` . Currently I went with the approach of adding UTs with the `getRemoteNodeName` in the latest commit. Please check.